chips/stm32f4xx/src/clocks/pll.rs

```diff
@@ -202,6 +206,29 @@ impl<'a, PllConstants: clock_constants::PllConstants> Pll<'a, PllConstants> {
         unreachable!("The previous for loop should always return");
     }
 
+    /// Get the PLL source clock frequency in MHz
+    ///
+    /// # Returns
+    ///
+    /// + [HSI_FREQUENCY_MHZ]: if the pll source is HSI
+    /// + HSE frequency if the pll source is HSE
+    fn get_pll_source_frequency(&self) -> usize {
+        if self.rcc.is_hsi_clock_system_clock() {
+            HSI_FREQUENCY_MHZ
+        } else {
+            self.rcc.get_hse_frequency().unwrap()
+        }
+    }
+
+    /// Set the PLL source clock according to the system clock
```

Comment:
Not sure if we should use documentation comments for private functions.

Reply:
I think if rustdoc is ok with it then it's fine.

Reply:
Agreed. Two benefits: (1) just consistency, and a good template for more documentation. (2) should the function every become public and picked up by rustdoc, the documentation is already in the right format. This is great.